### ISSUE TITLE
1.13 Support

### ISSUE THREAD

**Issue by ShaunMack** [mcedit/mcedit2#420]
Is 1.13 planned to be supported?

**Comment by Zitchas** [comment]
Given the significant changes and the likelyhood of people creating new worlds (where they may want to import their old stuff), I certainly hope so. This is one of the main tools I use with Minecraft. :)

**Comment by DawnTreader** [comment]
@dvaughan70 Actually, if you follow this [link ](https://github.com/mcedit/mcedit2/releases)you will notice that 13 days ago someone put up new compiled code. it can actually open a world and starts to draw a map, but crashes when you try to move the mouse to do anything. :(

but at least something is happening. and i am in the same boat, i would love to help but have no clue about where to start with this.

**Comment by dvaughan70** [comment]
yea there are people doing something andgod do i appreciate their efforts, but in general it seems alota folks have moved on...it looks like Mojang has totally rewritten the game code for Minecraft so most plugins and apps aren't the quick fix to update them to latest Minecraft release... It's going to possible mean a total rewrite of some apps and plugins.... If I knew anyone that could teach me this stuff I'd learn how. I got time....any tutors Wana elp me learn?

On Aug 27, 2018 6:18 PM, DawnTreader <notifications@github.com> wrote:

@dvaughan70<https://github.com/dvaughan70> Actually, if you follow this link <https://github.com/mcedit/mcedit2/releases> you will notice that 13 days ago someone put up new compiled code. it can actually open a world and starts to draw a map, but crashes when you try to move the mouse to do anything. :(

but at least something is happening. and i am in the same boat, i would love to help but have no clue about where to start with this.

**Comment by DmitryRendov** [comment]
just one this I'd like to ask you - please dont waste your time to support worlds migration (from 1.12 to 1.13.x) Forget about backward compatibility - just release new one and move forward.
thank you and looking forward...

**Comment by DawnTreader** [comment]
I agree with DmitryRendov, if it is easier to build a version that handles 1.13 but isn't backwards compatible then build it. Trying to keep compatibility isn't worth it when the game itself is upgrading the worlds with a warning that you should back them up and they won't work in 1.12 after you open them in 1.13.

I know that might mean a little pain for some users for a while, but at least the MCEdit that handled 1.12 fine will still be available to them. It will also mean that we can't open 1.12 worlds without first opening them in minecraft, but thats a little price to pay to have a working 1.13 MCEdit.

**Comment by dvaughan70** [comment]
To add a spawn schematic to a 1.13 world i had to make a 1.12 world add the schematic then transfer the world file to 1.13 server and it updates the rest of the world to1.13

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by dowen01** [comment]
Any news?

**Comment by StarterX4** [comment]
+
